packages/teleport/src/components/BannerList/Banner.test.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { render, fireEvent, screen } from 'design/utils/testing';
+
+import { Banner } from './Banner';
+
+describe('components/BannerList/Banner', () => {
+  it('displays the supplied message', () => {
+    const msg = 'I am a banner';
+    render(
+      <Banner message={msg} severity={0} id="test-banner" onClose={() => {}} />
+    );
+    expect(screen.getByText(msg)).toBeInTheDocument();
+  });
+
+  function assertComputedStyle(
+    win,
+    doc,
+    Component,
+    props,
+    styleName,
+    styleValue
+  ) {
+    const componentClass = Component(props).type.styledComponentId;
+    const componentElement = doc.getElementsByClassName(componentClass);
+    const style = win.getComputedStyle(componentElement[0]);
+    expect(style[styleName]).toBe(styleValue);
+  }
+
+  it('renders an info banner for under severity 5', async () => {
+    await Promise.all(
```

Comment:
Why we are using Promise.all here? 
```ts
    severity => {
        const bannerProps = {
          message: 'I am steve banner',
          severity,
          id: 'test-banner',
          onClose: () => {},
        };

        const { unmount } = render(<Banner {...bannerProps} />);
        expect(screen.getByRole('icon')).toHaveClass('icon-info_outline');

        assertComputedStyle(
          window,
          document,
          Banner,
          bannerProps,
          'backgroundColor',
          'rgb(3, 155, 229)'
        );

        unmount();
      }```
is a synchronous function so the whole `Promise.all` will execute callbacks sequentially

I think we should just check the severity border values or use https://jestjs.io/docs/api#each

Reply:
Yeah I wasn't concerned about the sequential nature as it's a simple test but using `each` made sense, so updated. I just wish it could update the test label per iteration 😄